docs/xdr/features/detect/rules_catalog.md

```diff
@@ -109,10 +109,30 @@ You can click on the name of a rule to display additional details, such as, but
 - Associated Data Sources
 - Known False Positives
 - The actual detection logic (the pattern)
+- Alert filters
 - [Similarity strategy](../../investigate/alerts/#similarity-strategies) for the produced alerts
 
 ![rule details](/assets/operation_center/rules_catalog/rule_details2.png)
 
+### Alert filters
+
+In the section `Alert filters`, you can consult the list of alert filters that are currently enabled in the `Àctives` tab. The number of alert `muted` in the `last 30 days` is displayed for each alert filter to help you audit its effectiveness.
```

Comment:
@StephKll3c not sure if it has been done this way on back end side (last 30 days for telemetry)

Reply:
@Sengthay I think there is an accent on the uppercase A of "Actives" that should be removed

Reply:
Indeed, there is no notion of 'in the last 30 days'.